Code/GraphMol/AddHs.cpp

```diff
@@ -409,6 +422,47 @@ void addHs(RWMol &mol, bool explicitOnly, bool addCoords,
     }
     // update the atom's derived properties (valence count, etc.)
     newAt->updatePropertyCache();
+
```

Comment:
Not critical, but it seems like this code should be in a separate function (in an anonymous namespace).